pkg/db/src/implementations/version-1.1/note.ts

```diff
@@ -240,7 +240,13 @@ class Note implements NoteInterface {
 		}, this.#initialized);
 	}
 
-	updateTransaction({ isbn, quantity, warehouseId }: PickPartial<VolumeStock, "warehouseId">): Promise<NoteInterface> {
+	updateTransaction({
+		match: { isbn, quantity, warehouseId },
+		update
+	}: {
+		match: PickPartial<VolumeStock, "warehouseId">;
+		update: VolumeStock;
+	}): Promise<NoteInterface> {
```

Comment:
Can you apply that signature, like we've discussed (adjusted with partial warehouseId):
```typescript
updateTransaction(match: PickPartial<Omit<VolumeStock, "quantity">, "warehouseId">, update: VolumeStock): Promise<NoteInterface>
```

Reply:
It's less messy to consume, as well as less code in the declaration